fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/ClientRepository.java

```diff
@@ -21,6 +21,6 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 
-abstract interface ClientRepository extends JpaRepository<Client, Long>, JpaSpecificationExecutor<Client> {
+public abstract interface ClientRepository extends JpaRepository<Client, Long>, JpaSpecificationExecutor<Client> {
```

Comment:
we are not exposing ClientRepository. We need to make use of ClientRepositoryWrapper